Jenkinsfile

```diff
@@ -36,6 +37,10 @@ pipeline {
         }
       }
 
+      environment {
+        GIT_BRANCH = "${CURRENT_BRANCH}"
```

Comment:
Why does this need to be set?  I see it in the build flags, but clarify the use?

Reply:
> Jenkins runs this step in a detacted head state, so the `GIT_BRANCH`
needs to be exposed to the build container in order for the coveralls
coverage report to link to the proper branch.

Right now the coverage doesn't post for the branch (https://coveralls.io/repos/318162/builds), this _should_ fix it. We'll be able to tell once this PR can pass tests.

Reply:
ah, nice, thanks